### ISSUE TITLE
Add "Scope" to About page

### ISSUE THREAD

**Issue by labarba** [openjournals/joss#582]
The scope of the journal should be made clear on the [About](https://joss.theoj.org/about) page. For example, the statement that

> Minor ‘utility’ packages, including ‘thin’ API clients, are not acceptable.

is hidden all the way down the page on [Submitting a paper to JOSS](https://joss.readthedocs.io/en/latest/submitting.html), in our documentation site. We are getting more and more submissions that fall in this category, and authors may not be aware of the eligibility limitations when they submit. We need to make it plainly clear.

**Comment by danielskatz** [comment]
I agree - I just had to find it to quote it in a rejection, and it took me a little bit of effort to find.

@labarba - why don't you create a PR for this?

**Comment by danielskatz** [comment]
Sorry, I now see that @xuanxu did this already

**Comment by labarba** [comment]
Deadlines. Created issue to remember, but could not allocate time now.

**Comment by arfon** [comment]
I've taken a pass at this in https://github.com/openjournals/joss/commit/d26e02936aea1edca664f8197c1dedd489f0e94c which is visible at https://joss.theoj.org/about#submitting.

Feel free to suggest modifications :-)

**Comment by labarba** [comment]
Well, it's fine, but it's not what I had in mind. You've copied one the "Submission requirements," listing all details, like you need a `paper.md`, and be a contributor to submit. What I had in mind was a section titled "Scope" that only addressed the _type of software_ that is eligible. I feel the details on how to submit should remain in the documentation site.

**Comment by arfon** [comment]
Sorry, was trying to move to fast :-)

How about this? https://github.com/openjournals/joss/pull/585
